### PR TITLE
[T02]: display the first 100 jobs

### PR DESCRIPTION
**Pull request by ChienVan01**
Story:
Link code feature: https://github.com/fabatek/onboarding-job-board-frontend/tree/feature/chientv/t02_display_the_first_100_jobs
Link API to see specific job totals: https://6239311d043817a543df0a3f.mockapi.io/api/jobs
Link deloy: https://delirious-seat.surge.sh/

Notes

1. Edit JobList componenet: Map data from API to view

Result
image
image
image

### CODE REVIEW

**Comment by ChienVan01** [comment]
> 

done

File: src/components/jobList/JobList.tsx [hunk 1]
```diff
@@ -1,7 +1,36 @@
...
-  return <div className='text-center'>JobList</div>;
-};
+  const jobState = useSelector((state: RootStore) => state.job);
+
+  return (
+    <div className='d-flex flex-wrap'>
+      {jobState.job?.map((i) => {
+        return (
+          <>
+            <div className='card w-25 p-3 '>
```
Comment (ducdmfaba) [inline src/components/jobList/JobList.tsx:15]: Missing key

**Review by ducdmfaba — CHANGES_REQUESTED** [review]
image

- Should have padding for section
- The item is too big
- Between items should have a margin

**Comment by ChienVan01** [comment]
> image
> 
> * Should have padding for section
> * The item is too big
> * Between items should have a margin

done

**Comment by ducdmfaba** [comment]
Is it changed? I see nothing

**Comment by ChienVan01** [comment]
> Is it changed? I see nothing

Sorry, I have updated again!

File: src/components/header/header.scss [hunk 2]
```diff
@@ -2,3 +2,19 @@
   padding-left: 150px !important;
   padding-right: 150px !important;
 }
+
+
+@media only screen and (max-width: 768px) {
+  .header {
+    padding-top: 20px !important;
+    padding-left: 50px !important;
+    padding-right: 50px !important;
```
Comment (ducdmfaba) [inline src/components/header/header.scss:11]: It should be `padding: 20px 50px 0 50px;`

File: src/components/jobList/jobList.scss [hunk 3]
```diff
@@ -5,3 +5,21 @@
...
+  .job__list--item{
+    width: 300px;
+  }
+}
+
+@media only screen and (max-width: 390px) {
+  .search__container {
+    padding-top: 0px !important;
+    padding-left: 0px !important;
+    padding-right: 0px !important;
```
Comment (ducdmfaba) [inline src/components/jobList/jobList.scss:23]: Same above.

File: src/components/seachContainer/searchContainer.scss [hunk 4]
```diff
@@ -3,3 +3,21 @@
...
+    padding-right: 100px !important;
+  }
+
+}
+
+@media only screen and (max-width: 390px) {
+  .search__container {
+    padding-top: 0px !important;
+    padding-left: 0px !important;
+    padding-right: 0px !important;
```
Comment (ducdmfaba) [inline src/components/seachContainer/searchContainer.scss:21]: Same above

**Review by ducdmfaba — CHANGES_REQUESTED** [review]

**Review by ducdmfaba — APPROVED** [review]

**Pull request merged**